pages/docs/guides/pause-functions.mdx

```diff
@@ -0,0 +1,59 @@
+export const description = 'Learn how to pause an Inngest function.'
+
+# Pausing functions
+
+Inngest allows you to pause a function indefinitely. This is a powerful feature that can be useful, for example, if you are planning a maintenance window or if a user reports a bug and you want to stop processing events until you've fixed it.
+
+## When a function is paused
+
+It's important to understand what happens when a function is paused.
+
+**No data will be lost.** Inngest will continue receiving and storing your events, but the events will not trigger the paused function. The function will be marked as "skipped" on the event's page in Inngest Cloud.
+
+**You can resume the function at any time.** No deployment or sync is required to resume your function. After you resume the function, new events will trigger it as usual.
+
+**Events received while the function was paused will not be reprocessed automatically** after you resume the function. Use [Replay](/docs/platform/replay) to process events that were received while the function was paused.
+
+**Paused functions do not count toward your plan's concurrency limit.** Note that events received while a function is paused are still subject to your plan's history limit.
+
+## How to pause a function
+
+Navigate to the function's page in Inngest Cloud, and click the Pause button near the top right corner:
+
+![Pause button in Inngest Cloud.](/assets/docs/guides/pause-functions/pause-btn.png)
+
+### Handling running invocations
+
+When you pause a function, no new events will trigger it. You can choose what to do with any currently-running invocations of the function:
+
+- **"Pause immediately, then cancel after 7 days:"** No further steps will be executed while the function is paused. If you resume the function within 7 days, these invocations will continue with the next step. Otherwise, they will be canceled. (This is the default behavior.)
+- **"Cancel immediately:"** All currently-running invocations of the function will be canceled. They will not continue running or restart when you resume the function.
+
+In both cases, all running invocations will complete whatever step they are currently running before being paused or canceled; Inngest cannot interrupt your function mid-step.
```

Comment:
```suggestion
In both cases, all running invocations will complete their current step before being paused or canceled. Inngest cannot interrupt your function mid-step.
```

I thought to make this paragraph a little bit more concise, see if it works for you!